docs/devguide/migrate-dashboards.asciidoc

```diff
@@ -0,0 +1,43 @@
+== Migrating dashboards from Kibana 5.x to 6.x
+
+Starting with Kibana 6.0, the format of the dashboards changes and it's not compatible with the Kibana 5.x dashboards.
+
+Each module keeps the Kibana 5.x dashboards under a 5.x directory, and under the default directory for Kibana 6.0 or greater. 
+
+The 5.x directory should contain the following directories:
+- search
+- visualization
+- dashboard
+- index-pattern
+
+The default directory should contain the following directories:
+- dashboard
+- index-pattern
+
+To collect all the dashboards per module to the global `_meta/kibana` director, you would need to run `make collect`.
+The index-pattern directory is generated from fields.yml by the `make update` command. 
+
+To migrate your Kibana 5.x dashboards to Kibana 6.0 and above, you can import the dashboards into Kibana 5.6 and then
+exporting them using Beats 6.0 version.
+
+1. Start Kibana 5.6
+2. Import Kibana 5.x dashboards using Beats 6.0 version. Before importing the dashboards, make sure you run `make
```

Comment:
First part of the sentence makes me think the import happens in Kibana. I'd switch things around a little and possibly have `make update` as a discrete step. I also think it helps to say why (TBH, I'm not sure here, so I've guessed). Also remove the $ from commands...users complain because they can't copy/paste as easily).

```
2. From the Beat directory, run `make update` to generate index patterns:

[source,shell]
-----------------
make update
-----------------

2. Run the `setup` command to import the Kibana 5.x dashboards into Kibana 6.0. For example:

[source,shell]
-----------------
./metricbeat setup -E setup.dashboards.directory=_meta/kibana
-----------------

3. Run the `export_dashboards` tool to export version 6.0 dashboards. For example:

[source,shell]
-----------------
cd beats/dev-tools/cmd/dashboards
./export_dashboards -dashboards Metricbeat-Rabbitmq
-----------------
```

I've added "for example" here because, believe it or not, users will copy/paste the command for metricbeat and wonder why it's not working for filebeat.